### PR TITLE
481 lang is deprecated remove references to it in the docs

### PR DESCRIPTION
**Pull request by anders314159**

### CODE REVIEW

**Comment by anders314159** [comment]
also removed a few references to docker: in favor of jobtype_extra:

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -668,7 +668,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - One job type can be installed in multiple versions at the same time.
   Users have to pick one of these versions and specify it in the manifest of their fatman,
-  eg. `lang: python3:2.4.0`.
+  eg. `jobtype: python3:2.4.0`.
   Version of a job type is now required in the `lang` field of Manifest in a manner `name:version`.
```
Comment (iszulcdeepsense) [inline docs/CHANGELOG.md:672]:
```suggestion
  Version of a job type is now required in the `jobtype` field of Manifest in a manner `name:version`.
```

File: docs/deployment/remote-docker-gateway.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ infrastructure_targets:
     remote_gateway_url: 'http://1.2.3.4:7105/pub'
     remote_gateway_token: '5tr0nG_PA55VoRD'
 
-docker: 
+jobtype_extra:
```
Comment (iszulcdeepsense) [inline docs/deployment/remote-docker-gateway.md:47]: This YAML is not the job's manifest, but the configuration of the plugin, so in this case this field is valid:
```suggestion
docker:
```

File: docs/deployment/remote-kubernetes-gateway.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ infrastructure_targets:
     remote_gateway_token: '5tr0nG_PA55VoRD'
     job_k8s_namespace: 'racetrack'
 
-docker: 
+jobtype_extra:
```
Comment (iszulcdeepsense) [inline docs/deployment/remote-kubernetes-gateway.md:47]: Same here:
```suggestion
docker:
```

File: utils/standalone-wizard/remote-docker/plugin-config.yaml [hunk 4]
```diff
@@ -3,7 +3,7 @@ infrastructure_targets:
     remote_gateway_url: 'http://${public_ip}:7105/pub'
     remote_gateway_token: '${remote_gateway_token}'
 
-docker:
+jobtype_extra:
```
Comment (iszulcdeepsense) [inline utils/standalone-wizard/remote-docker/plugin-config.yaml:6]: Same here. It's a plugin's config

File: utils/standalone-wizard/remote-kubernetes/plugin-config.yaml [hunk 5]
```diff
@@ -4,7 +4,7 @@ infrastructure_targets:
     remote_gateway_token: '${remote_gateway_token}'
     job_k8s_namespace: '${kubernetes_namespace}'
 
-docker:
+jobtype_extra:
```
Comment (iszulcdeepsense) [inline utils/standalone-wizard/remote-kubernetes/plugin-config.yaml:7]: Same here. It's a plugin's config

**Review by iszulcdeepsense — APPROVED** [review]

**Pull request merged**